core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -57,66 +57,77 @@ var LearnerDashboardPage = function() {
+      'Play Later Explorations Tab', playLaterSection);
   };
 
   this.navigateToCompletedSection = async function() {
     await waitFor.elementToBeClickable(
       completedSection, 'Completed tab takes too long to be clickable');
-    await completedSection.click();
+    await action.click(
+      'Completed Tab', completedSection);
   };
 
   this.navigateToInCompleteSection = async function() {
     await waitFor.elementToBeClickable(
       incompleteSection, 'In Progress tab takes too long to be clickable');
-    await incompleteSection.click();
+    await action.click(
+      'In Progress Tab', incompleteSection);
   };
 
   this.navigateToIncompleteCollectionsSection = async function() {
     await waitFor.elementToBeClickable(
       incompleteCollectionsSection,
-      'Incomplete Collection Section tab takes too long to be clickable');
-    await incompleteCollectionsSection.click();
+      'In Progress Collections tab takes too long to be clickable');
+    await action.click(
+      'In Progress Collections Tab', incompleteCollectionsSection);
   };
 
   this.navigateToIncompleteExplorationsSection = async function() {
     await waitFor.elementToBeClickable(
       incompleteExplorationsSection,
-      'Incomplete Collection Section tab takes too long to be clickable');
-    await incompleteExplorationsSection.click();
+      'In Progress Explorations tab takes too long to be clickable');
+    await action.click(
+      'In Progress Explorations Tab',
+      incompleteExplorationsSection);
   };
 
   this.navigateToCompletedCollectionsSection = async function() {
     await waitFor.elementToBeClickable(
       completedCollectionsSection,
-      'Completed Collection Section tab takes too long to be clickable');
-    await completedCollectionsSection.click();
+      'Completed Collection tab takes too long to be clickable');
+    await action.click(
+      'Completed Collections Tab', completedCollectionsSection);
   };
 
   this.navigateToCompletedExplorationsSection = async function() {
     await waitFor.elementToBeClickable(
       completedExplorationsSection,
-      'Completed Collection Section tab takes too long to be clickable');
-    await completedExplorationsSection.click();
+      'Completed Exploration tab takes too long to be clickable');
+    await action.click(
+      'Completed Explorations Tab', completedExplorationsSection);
   };
 
   this.navigateToFeedbackSection = async function() {
     await waitFor.elementToBeClickable(
       feedbackSection, 'Feedback Section tab takes too long to be clickable');
-    await feedbackSection.click();
+    await action.click(
+      'Feedback Tab', feedbackSection);
   };
 
   this.navigateToFeedbackThread = async function() {
     await waitFor.elementToBeClickable(
       feedbackThread, 'Feedback Thread tab takes too long to be clickable');
-    await feedbackThread.click();
+    await action.click(
+      'Feedback Thread Tab', feedbackThread);
```

Comment:
Ditto as above (here and wherever possible following max 80 chars in a line)

Reply:
Done